cmd/queue/main.go

```diff
@@ -205,12 +183,17 @@ func handler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, handler http.H
 				http.Error(w, fmt.Sprintf(badProbeTemplate, ph), http.StatusBadRequest)
 				return
 			}
-			if probeUserContainer() {
-				// Respond with the name of the component handling the request.
-				w.Write([]byte(queue.Name))
+			if prober != nil {
```

Comment:
Maybe in a separate PR: Is there a reason why we don't return the state from `healthState` here?  Seems unnecessarily redundant to probe on this path 🤔 

@greghaynes do you need that for your "direct to ip" work?

Reply:
That seems like an excellent suggestion. 👍